app/models/ability.rb

```diff
@@ -0,0 +1,13 @@
+class Ability
```

Comment:
regarding ability.rb moving out — i get that people can customize abilities by making roles, but don’t we want them to have full control over abilities if they want it? so maybe it’s worth putting ability.rb back in the file